.gitignore

```diff
@@ -22,3 +22,5 @@ erl_crash.dump
 # Ignore package tarball (built via "mix hex.build").
 elatrix-*.tar
 
+# VSCode settings
+.vscode
```

Comment:
New line at the end of file is good though to have less changes when adding smth

Reply:
@MKenin wdyt?